tests/unit/bluetooth/at/prj.conf

```diff
@@ -3,3 +3,4 @@ CONFIG_BLUETOOTH_BREDR=y
 CONFIG_BLUETOOTH_HFP_HF=y
 CONFIG_NET_BUF=y
 CONFIG_ZTEST=y
+CONFIG_SERIAL=y
```

Comment:
It is. UART is just one of possible ways of connecting a Bluetooth controller (although it's the most common one). We e.g. have a SPI driver as well. Anyway, the dependency on HCI transport should come from the chosen HCI driver. Since this is a BR/EDR test we could of course decides that it's purely for qemu (which is the only platform where we can test BR/EDR at the moment) and with qemu UART is the only currently supported way of interfacing with a Bluetooth controller.